anchore-ci/common_tasks

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env bash
+
+############################################################
+# Common CI tasks. These are intended to be reusable across
+# projects, and easily overriden.
+############################################################
+
+clean() {
+  TEST_IMAGE_NAME="${1:?'Missing required parameter: TEST_IMAGE_NAME.'}"
+
+  print_colorized WARN "Deleting virtualenv, cache, build and testing temp data."; echo
+
+  continue_prompt
+  rm -rf venv ./*.egg-info dist build .tox
```

Comment:
If this is going to delete the venv it should accept the venv name in a VENV parameter variable

Reply:
Yepper